repos/system_upgrade/common/actors/scanfilestocopyintotargetuserspace/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.scanfilestocopyintotargetuserspace import scan_files_to_copy
+from leapp.models import CopyFile
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class ScanFilesToCopyIntoTargetUserspace(Actor):
```

Comment:
```suggestion
class ScanFilesToBeCopiedIntoTargetUserspace(Actor):
```

I am not insisting on this change as it will make the name of the object even longer.

Reply:
I am for  this change as the suggested name outlines the purpose of this actor even better.

Reply:
Suggested change applied in the newest version.